srcpkgs/66-boot/INSTALL.msg

```diff
@@ -0,0 +1 @@
+To boot with 66, simply add the argument `init=/usr/bin/66' to your kernel commandline.
```

Comment:
This shouldn't be a INSTALL message.

Reply:
I will create a doc for docs.voidlinux.org